src/main/java/com/cryptomorin/xseries/XMaterial.java

```diff
@@ -2151,12 +2120,29 @@ private boolean isDuplicated() {
          *
          * @since 1.0.0
          */
-        private static final int VERSION = Integer.parseInt(getMajorVersion(Bukkit.getVersion()).substring(2));
+        private static final int VERSION = parseVersion(Bukkit.getVersion());
         /**
          * Cached result if the server version is after the v1.13 flattening update.
          *
          * @since 3.0.0
          */
         private static final boolean ISFLAT = supports(13);
+
+        /**
+         * Gets the exact major version (..., 9, 10, ..., 14)
+         *
+         * @param version Supports the version as provided by {@link Bukkit#getVersion()}
+         *
+         * @return the exact major version.
+         * @see #VERSION
+         * @since 8.5.0
+         */
+        private static int parseVersion(String version) {
+            Matcher matcher = Pattern.compile("MC: \\d\\.(\\d+)").matcher(version);
+            if (matcher.find()) {
+                return Integer.parseInt(matcher.group(1));
+            }
+            return -1;
```

Comment:
Just blindly returning a negative number and letting the program execute normally is definitely not desirable. If version cannot be detected, the program should fail (by throwing an **IllegalArgumentException**)

Reply:
Fair point, I've added an IllegalArgumentException